drape/drape_tests/harfbuzz_shaping_test.cpp

```diff
@@ -0,0 +1,28 @@
+#include "testing/testing.hpp"
+#include "drape/harfbuzz_shaping.hpp"
+
+namespace harfbuzz_shaping
```

Comment:
_tests?

Reply:
I thought that it would be a good idea to avoid `using harfbuzz_shaping;` in tests by including tests in the same namespace. WDYT?

Reply:
as you wish

Reply:
No approval means... what? )